frontend/src/metabase/entities/containers/EntityForm.jsx

```diff
@@ -1,11 +1,12 @@
 /* eslint-disable react/prop-types */
-import { Component } from "react";
+import { Component, lazy, Suspense } from "react";
 import { t } from "ttag";
 
-import Form from "metabase/containers/FormikForm";
 import ModalContent from "metabase/components/ModalContent";
+import LoadingAndErrorWrapper from "metabase/components/LoadingAndErrorWrapper";
 
 import entityType from "./EntityType";
+const Form = lazy(() => import("metabase/containers/FormikForm"));
```

Comment:
Nit: it's easier to find usages when the reference name matches the file name

```suggestion
const FormikForm = lazy(() => import("metabase/containers/FormikForm"));
```